tests/by-util/test_wc.rs

```diff
@@ -168,3 +170,58 @@ fn test_file_one_long_word() {
         .run()
         .stdout_is("    1     1 10001 10001 10000 onelongword.txt\n");
 }
+
+/// Test that the number of bytes in the file dictate the display width.
+///
+/// The width in digits of any count is the width in digits of the
+/// number of bytes in the file, regardless of whether the number of
+/// bytes are displayed.
+#[test]
+fn test_file_bytes_dictate_width() {
+    // This file has 10,001 bytes. Five digits are required to
+    // represent that. Even though the number of lines is 1 and the
+    // number of words is 0, each of those counts is formatted with
+    // five characters, filled with whitespace.
+    new_ucmd!()
+        .args(&["-lw", "onelongemptyline.txt"])
+        .run()
+        .stdout_is("    1     0 onelongemptyline.txt\n");
+
+    // This file has zero bytes. Only one digit is required to
+    // represent that.
+    new_ucmd!()
+        .args(&["-lw", "emptyfile.txt"])
+        .run()
+        .stdout_is("0 0 emptyfile.txt\n");
+}
+
+/// Test that getting counts from a directory is an error.
+#[test]
+fn test_read_from_directory_error() {
+    // TODO To match GNU `wc`, the `stderr` should be:
+    //
+    //     "wc: .: Is a directory\n"
+    //
+    // TODO To match GNU `wc`, the `stdout` should be:
+    //
+    //     "      0       0       0 .\n"
+    //
+    new_ucmd!()
+        .args(&["."])
+        .run()
+        .stderr_is("wc: error: Expected a file, found directory .\n")
+        .stdout_is("0 0 0 .\n");
+}
+
+/// Test that getting counts from nonexistent file is an error.
+#[test]
+fn test_read_from_nonexistent_file() {
+    // TODO To match GNU `wc`, the `stderr` should be:
+    //
+    //     "wc: bogusfile: No such file or directory\n"
+    //
+    new_ucmd!()
+        .args(&["bogusfile"])
+        .run()
```

Comment:
If the test is expected to fail, please use `fails()` instead of `run()` (checks if error code is 1).
For the error messages I would just test like this: `stderr_contains("bogusfile: No such file or directory")`.
Because the prefix is subject of change if/when the macros are changes.

Reply:
Thanks for the advice, I will make those changes.